TableSchemer/TableScheme.swift

```diff
@@ -51,6 +51,11 @@ public class TableScheme: NSObject {
 
         tableView.dataSource = self
         tableView.delegate = self
+        if #available(iOS 11.0, *) {
```

Comment:
I think we'll want to put this behind a flag in the initializer that defaults to the same behavior prior to this patch. Otherwise we'll end up changing the behavior on others users of the library when they may not be expecting it.